requirements.txt

```diff
@@ -0,0 +1 @@
+nose==1.3.7
```

Comment:
This file tells pretty much everyone who looks at it what third party packages are required in order to run the code here. In our case, we need `nose` because it has `nosetests`. Which is way better at running tests.